src/features/login/Login.tsx

```diff
@@ -0,0 +1,81 @@
+import React, {useState} from 'react';
+import {View, Text} from 'react-native';
+import {connect} from 'react-redux';
+import {Dispatch} from 'redux';
+
+import {loginStyles} from './loginScreenStyles';
+import {MyButton} from '../../components/button/Button';
+import {InputText} from '../../components/input/Input';
+import {testData} from '../../services/authService';
+
+export interface Props {
+  placeholder: string;
+  navigation: {
+    navigate: (route: string) => void;
+    reset: (routes: {}) => void;
+  };
+}
+
+const login: React.FC<Props> = ({navigation}) => {
+  const [inputEmail, setInputEmail] = useState('');
+  const [inputPassword, setInputPassword] = useState('');
+
+  const [validEmail, setValidEmail] = useState(true);
+  const [validPassword, setValidPassword] = useState(true);
+
+  const emailReg =
+    /^[a-zA-Z0-9.! #$%&'*+/=? ^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+
+  const validUser =
+    inputEmail === testData.email && inputPassword === testData.password;
+
+  const tryEmail = () => {
+    if (emailReg.test(inputEmail) == true) {
+      return setValidEmail(true);
+    }
+    return setValidEmail(false);
```

Comment:
how we can improve this statement?
we calling one func with diff arg based on 1 statement ...